corehq/apps/hqwebapp/tasks.py

```diff
@@ -43,8 +44,8 @@ def mark_subevent_gateway_error(messaging_event_id, error, retrying=False):
 
 @task(serializer='pickle', queue="email_queue",
       bind=True, default_retry_delay=15 * 60, max_retries=10, acks_late=True)
-def send_mail_async(self, subject, message, from_email, recipient_list,
-                    messaging_event_id=None, domain=None):
+def send_mail_async(self, subject, message, configuration: EmailConfigurationManager,
+                    recipient_list, messaging_event_id=None, domain: str = None):
```

Comment:
I think our current policy to not to use type hints

Reply:
I didn't know that, I personally find it helpful but I would be fine with removing it. The only case is for `configuration` -is it more standard to just add a comment that `configuration` is a `EmailConfigurationManager`? Otherwise, it'll be hard to figure out just by reading the code.